include/nuttx/compiler.h

```diff
@@ -999,6 +999,13 @@
 
 #endif
 
+/* fallthrough */
+
+#define fallthrough do \
```

Comment:
fallthrough is very easy conflict with other code, why add this

Reply:
remove